src/main/java/com/stripe/model/BalanceTransaction.java

```diff
@@ -100,4 +100,15 @@ public static BalanceTransaction retrieve(String id, RequestOptions options)
     return request(RequestMethod.GET, url, null, BalanceTransaction.class, options);
   }
   // </editor-fold>
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class Fee extends ApiResource {
```

Comment:
This one is an `ApiResource`, is it expected that it was moved?

Reply:
It shouldn't be -- nested objects should all be `StripeObject`s, not `ApiResource`s.